wsf_scraping/spiders/nice_spider.py

```diff
@@ -15,6 +15,10 @@ class NiceSpider(scrapy.Spider):
         'JOBDIR': 'crawls/nice'
     }
 
+    def __init__(self, *args, **kwargs):
+        id = kwargs.get('uuid', '')
+        self.uuid = id
```

Comment:
should we keep running if there is not `uuid` or raise an exception?

Reply:
The uuid part is only a way to manage spiders in the API part. Running the spider without the API would be the only way to have a spider without uuid, and won't lead to any problems.